make/photon/prepare/utils/redis.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+from g import data_dir, REDIS_UID, REDIS_GID
+from utils.misc import prepare_config_dir
+
+redis_data_path = os.path.join(data_dir, 'redis')
+
+def prepare_redis(config_dict):
+    prepare_config_dir(redis_data_path)
```

Comment:
Generally after going over the changes in python, it seems better if we rename the `prepare_config_dir` to `prepare_dir` and make `uid` `gid` optional parms, so that there will be less repetitive code.